packages/api/src/graphql/page.ts

```diff
@@ -135,14 +137,79 @@ export const GraphQLPage = new GraphQLObjectType<Page, Context>({
     createdAt: {type: GraphQLNonNull(GraphQLDateTime)},
     modifiedAt: {type: GraphQLNonNull(GraphQLDateTime)},
 
-    draft: {type: GraphQLPageRevision},
-    published: {type: GraphQLPageRevision},
-    pending: {type: GraphQLPageRevision},
+    draft: {
```

Comment:
Why are you writing the resolver for `darft`, `published`, `pending` and `latest` if you can write it just once in the `GraphQLPageRevision`?

Reply:
@roosnic1 fixed.